woocommerce/woocommerce
# Add "launch your store" task

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/45108.

Add  "launch your store" task to the onboarding task list and update the task list header style to match the design.


<img src="https://github.com/woocommerce/woocommerce/assets/4344253/ed86700d-4c7f-4af3-971e-64b24f4ad8d8" width="700px" />

Figma: 0BWMerqRR1f468EwC7g5tg-fi-835_27969





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a fresh site
2. Install WooCommerce Beta Tester.
3. Run `wp option update launch-status unlaunched`
4. Skip Core Profiler
5. Go to `WooCommerce > Home`
6. Observe that  "launch your store" task is not shown.
7. Go to `/wp-admin/tools.php?page=woocommerce-admin-test-helper`
8. Toggle the feature flag for `launch-your-store` to true
9. Go to `WooCommerce > Home`
10. Observe that the "launch your store" task is shown.
11. Complete all tasks except the "launch your store" task
12. Confirm task header matches the design
13. Run `wp option update launch-status launched`
14. Confirm tasklist is completed.

![Image](https://github.com/woocommerce/woocommerce/assets/4344253/a3bcd9e6-2b92-4a5d-ae8a-5bbf3d2b8c40)





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
